include/OpenSpaceToolkit/Astrodynamics/Trajectory/Segment.hpp

```diff
@@ -118,6 +118,11 @@ class Segment
 
     const Shared<EventCondition>& accessEventCondition() const;
```

Comment:
Sorry, this is not part of the change. Noticed it when comparing to the other you added, but should this be:

```suggestion
    const Shared<const EventCondition>& accessEventCondition() const;
```
?

if we don't want the event conditions associated with the segment to be modified

Reply:
Ah shit you're right, I don't think I need to add this method as the EventCondition is not const. I'll just delete my method.